project1/ConstantBSProcess.cpp

```diff
@@ -0,0 +1,66 @@
+#include <blackscholesprocess.hpp>
+#include <ConstantBSProcess.h>
+
```

Comment:
You defined `ConstantBSProcess` in namespace QuantLib (see the hpp) so the implementation also needs to be inside the same namespace.
```suggestion
namespace QuantLib {
```
You'll also need to close the namespace at the end of the file.